alibi/explainers/integrated_gradients.py

```diff
@@ -833,7 +838,7 @@ def explain(self,
                                                                    baselines,
                                                                    self.model,
                                                                    self.layer,
-                                                                   self.orig_call,
+                                                                   self.orig_call,  # type: ignore # TODO: fix me
```

Comment:
From the type-checker point of view `self.orig_call` can still bet `None` even though we narrowed it down via `else` clause (following `if self.layer is None`).

Reply:
Mmn this one seems annoying...